core/model/models/network-config.js

```diff
@@ -115,7 +115,7 @@ exports.NetworkConfig = AbstractModel.specialize({
             iconComponentModule: {
                 id: 'ui/icons/network-configuration.reel'
             },
-            nameExpression: "'Network Configuration'"
```

Comment:
@mactanxin that should not be done here.
https://github.com/freenas/gui/blob/master/core/model/user-interface-descriptors/network-config-user-interface-descriptor.mjson#L7